test/sqlsmith/mzcompose.py

```diff
@@ -0,0 +1,175 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+import json
+import random
+import time
+from datetime import datetime
+from threading import Thread
+from typing import Any, Dict, FrozenSet, List, Tuple
+
+from materialize.mzcompose import Composition, Service, WorkflowArgumentParser
+from materialize.mzcompose.services import Materialized
+
+SERVICES = [
+    Materialized(),
+    Service(
+        "sqlsmith",
+        {
+            "mzbuild": "sqlsmith",
+        },
+    ),
+]
+
+
+# These are only errors which have no associated issues since they are not
+# considered product bugs, but SQLsmith generating bad queries. Use ci-regexp
+# in Github issues for actual product bugs.
+known_errors = [
+    "no connection to the server",  # Expected AFTER a crash, the query before this is interesting, not the ones after
+    "failed: Connection refused",  # Expected AFTER a crash, the query before this is interesting, not the ones after
+    'could not translate host name "materialized" to address: Name or service not known',  # Expected AFTER a crash, the query before this is interesting, not the ones after
```

Comment:
This needs to abort the CI job with a failure right away, so should not be on the `known_errors` list.

Reply:
Since we handle crashes now, they are just noise. The panic is more interesting.